bot/exts/christmas/hanukkah_embed.py

```diff
@@ -55,25 +56,26 @@ async def hanukkah_festival(self, ctx: commands.Context) -> None:
         month = str(today.month)
         year = str(today.year)
         embed = Embed()
-        embed.title = 'Hanukkah'
+        embed.title = "Hanukkah"
         embed.colour = Colours.blue
         if day in self.hanukkah_days and month in self.hanukkah_months and year in self.hanukkah_years:
             if int(day) == hanukkah_start_day:
                 now = datetime.datetime.utcnow()
-                now = str(now)
-                hours = int(now[11:13]) + 4  # using only hours
+                hours = now.hour + 4  # using only hours
                 hanukkah_start_hour = 18
                 if hours < hanukkah_start_hour:
                     embed.description = (f"Hanukkah hasnt started yet, "
                                          f"it will start in about {hanukkah_start_hour-hours} hour/s.")
```

Comment:
What do you think about adding spaces around the `-` in the f-strings here? It was a bit confusing to read right now

Reply:
Resolved in https://github.com/python-discord/sir-lancebot/pull/718/commits/dd20f59b9fb12288f286d479b5719697ab1a30f4.